app/Repositories/PostRepository.php

```diff
@@ -205,14 +204,6 @@ public function reviews(Post $post, $status, $comment = null, $admin = null)
         // Update the "counter cache" on the Post Campaign:
         $post->campaign->refreshCounts();
 
-        // If the Post has a School ID, upsert an ActionStat.
-        if ($post->school_id && $post->action_id) {
```

Comment:
Why were we checking for the `action_id` here? I noticed we omit that check above?

Reply:
I think this was maybe to safeguard for when posts are created with an `action` vs `action_id` -- but I'm not sure this was even necessary.